contracts/eip20/EIP20.sol

```diff
@@ -3,7 +3,7 @@ Implements EIP20 token standard: https://github.com/ethereum/EIPs/blob/master/EI
 .*/
 
 
-pragma solidity ^0.4.18;
+pragma solidity ^0.4.21;
```

Comment:
This isn't technically necessary, and not applying it to the other contract files feels inconsistent. 

I'd suggest either:

1. Upgrade all solidity file pragmas, and add the `emit` keyword. 
2. Don't change this pragma statement.

Reply:
My reasoning for this is to that people know what the up-to-date version is without having to consult Truffle. 

Adding the 'emit' keyword will break the CI afaik. But I like the latter. Will try that & see if it breaks it or not.

Upgrading all pragmas & adding emit